translator/freecell_translator/freecell_translator.py

```diff
@@ -20,28 +27,52 @@ def get_moves(self):
             src_card, dst_card = move
             cards_moved_vector, src_vector = get_source_card_vector(board, free_cells, src_card)
             dst_vector = get_dest_card_vector(board, dst_card)
-            move_vectors.append((cards_moved_vector, src_vector, dst_vector))
 
-        return move_vectors
+            move_id = self.all_moves[(cards_moved_vector, src_vector, dst_vector)]
+            move_vectors.append(move_id)
 
+        return move_vectors
 
+    # Our ml model takes one dimensional inputs
     def get_board(self):
         board, free_cells, heap = self.game.get_board()
 
         board = convert_board_to_ar_ohe(board)
         free_cells = convert_fc_to_ar_ohe(free_cells)
         heap = convert_heap_to_ar_ohe(heap)
 
-        return board, free_cells, heap
+        return np.concatenate((board, free_cells, heap)).tolist()
 
     def get_state(self):
         return self.game.get_state()
     
+    # To do
     def start_game(self):
-        self.game.start_game()
+        pass
```

Comment:
Mozna to juz zmienic, była zmiana w Freecell wedlug twoich uwag

Reply:
Fixed